Dockerfile

```diff
@@ -1,28 +1,46 @@
-FROM gradle:jdk-alpine
+# FROM gradle:jdk-alpine
 
-ARG SBT_VERSION=1.2.8
+### Taken from: https://github.com/mozilla/docker-sbt
+
+ARG OPENJDK_TAG=8u232
+FROM openjdk:${OPENJDK_TAG}
+
+ARG SBT_VERSION=1.4.5
+
+# Install sbt
+RUN \
+  mkdir /working/ && \
+  cd /working/ && \
+  curl -L -o sbt-$SBT_VERSION.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
+  dpkg -i sbt-$SBT_VERSION.deb && \
+  rm sbt-$SBT_VERSION.deb && \
+  apt-get update && \
+  apt-get install sbt && \
+  cd && \
+  rm -r /working/ && \
+  sbt sbtVersion
```

Comment:
Why not directly use the `mozilla/sbt` image?

```dockerfile
FROM mozilla/sbt:8u232_1.4.5
```

Reply:
tried to address both points, it should work now even better now :)